cmd/dep/testdata/harness_tests/init/gb/case1/final/Gopkg.toml

```diff
@@ -0,0 +1,10 @@
+
+[[constraint]]
+  name = "github.com/sdboyer/deptest"
+  source = "https://github.com/sdboyer/deptest"
+  version = "1.0.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Right now, we are seeing "branch = master" here because the revision is the tip of master (which `lookupVersionForLockedProject` identifies). I am thinking that if the revision wasn't the tip however, we'd still like the constraint to be master. If we end up adding logic for using the branch from gb, using an earlier revision from master would be one way to test that out.